examples/image_classification.ipynb

```diff
@@ -910,10 +910,10 @@
     }
```

Comment:
<p>Update for the last two paragraphs:</p><p> </p><p>To make sure we (1) resize to the appropriate size (2) use the appropriate image mean and standard deviation for the model architecture we are going to use, we instantiate what is called an image processor with the&nbsp;<code>AutoImageProcessor.from_pretrained</code>&nbsp;method.</p><p>This image processor is a minimal preprocessor that can be used to prepare images for inference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>